src/oracles/ChainlinkOracle.sol

```diff
@@ -74,24 +72,36 @@ contract ChainlinkOracle is IOracle, Ownable {
 
     /// @inheritdoc IOracle
     function price(address token) external view returns (bool success, uint256 priceX96) {
-        PriceData memory priceData = pricesInfo[token];
+        PriceData storage priceData = pricesInfo[token];
+        (
+            IAggregatorV3 feed,
+            uint48 heartbeat,
+            uint48 fallbackUpdatedAt,
+            uint256 fallbackPriceX96,
```

Comment:
The main idea of this trick was to load `fallbackPriceX96` only when necessary, i.e. when the chainlink is offline.
This approach lowers gas usage, as we have 1 SLOAD less on a most common execution branch.